### ISSUE TITLE
Easier Way to Get A Sharable URL

### ISSUE THREAD

**Issue by cycomachead** [jmoenig/Snap#338]
It would be great if we could present a shared project URL to the URL in an easy to see manner. The current way of copying the browser URL is quite confusing for many.

**Comment by cycomachead** [comment]
Partial fix with #624, but leaving this open. 

Some options to fix this are.
* show a message for 2s letting the user know to copy the URL (now that 624 is merged)
* popup a "real" html text field instead of a morphic one.
* fix C&P :P

**Comment by bromagosa** [comment]
Hey, we got this done in BeetleBlocks as a *save and share* button!

We can share the code.

**Comment by cycomachead** [comment]
> We can share the code.

I always know where to look for code samples. ;)

I want a "Save and Share" button too, see #714 
Even with that button, the problem that I've still seem with many students is that even if they share the project, they don't know to look at the URL or that the URL is even the thing they should be sharing.

**Comment by bromagosa** [comment]
Yep, that's pretty much solved by our save and share button too. It pops a system window with the URL in a text box so you can copy it straight away :)
